composer.json

```diff
@@ -19,7 +19,7 @@
         }
     },
     "require": {
-        "php": ">=5.5",
+        "php": ">=7.4",
```

Comment:
Could we have php 8.2 here instead? As our servers are running on 8.2 and up anyway.